hoverpibot:
  hardware_interface:
     loop_hz: 10 # hz
     joints:
       - left_wheel
       - right_wheel
